### PR TITLE
Add playstore command

### LINKED ISSUE #67: Google Play search command
https://github.com/facundoolano/google-play-scraper

### PR DESCRIPTION
**Pull request by BlueSlimee**
:man_shrugging: 

Closes #67

### CODE REVIEW

**Comment by Doges** [comment]
deal with standard:tm:

**Comment by BlueSlimee** [comment]
No problems with standard :thinking:

**Comment by BlueSlimee** [comment]
ok, i'm fixing the problem with package.json & package-lock

File: src/commands/utility/playstore.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      new StringParameter({ full: true, missingError: 'commands:playstore.noAppProvided' })
+    )
+  }
+  run ({ t, author, channel }, country, appName) {
+    channel.startTyping()
+    gplay.search({ term: appName, country: country, num: 8 }).then((apps) => {
+      const listEmbed = new SwitchbladeEmbed(author)
+        .setTitle(t('commands:playstore.results', { appName }))
+        .setDescription(`**${t('commands:playstore.sendNumber')}**\n\n\`[1]\` ${apps[0].title}\n\`[2]\` ${apps[1].title}\n\`[3]\` ${apps[2].title}\n\`[4]\` ${apps[3].title}\n\`[5]\` ${apps[4].title}\n\`[6]\` ${apps[5].title}\n\`[7]\` ${apps[6].title}\n\`[8]\` ${apps[7].title}`)
+      channel.send(listEmbed).then(async (msg) => {
```
Comment (perronosaurio) [inline src/commands/utility/playstore.js]: may use async?

Comment (perronosaurio) [reply src/commands/utility/playstore.js]: https://github.com/SwitchbladeBot/switchblade/wiki/Command-Guidelines#use-async-and-await-instead-of-then-whenever-possible-example

**Comment by BlueSlimee** [comment]
Ok, just a sec

File: src/commands/utility/playstore.js (hunk 1 above)
Comment (BlueSlimee) [reply src/commands/utility/playstore.js]: Ok, fixed.

**Comment by BlueSlimee** [comment]
fuck github :facepalm:

**Pull request closed without merging**